cristin-import/src/main/java/no/unit/nva/cristin/lambda/CristinEntryEventConsumer.java

```diff
@@ -57,6 +58,10 @@
     public static final String ERRORS_FOLDER = "errors";
 
     private static final Logger logger = LoggerFactory.getLogger(CristinEntryEventConsumer.class);
+    private static final Clock CLOCK = Clock.systemDefaultZone();
+    //imported cristin entries have for now no owner. Calling the Person proxy will only increase the cost of import.
+    private static final HttpClient CLIENT_AVOIDING_REQUESTS_FOR_RESOURCES_WITHOUT_OWNERS = new FakeHttpClient();
```

Comment:
Using a Fake Http Client when we are imporing data from Cristin because these entries have a placeholder for owner. There is no point invoking 2 million times the Person proxy for an imaginary user. In addition, we want all resources to be visible to us (Unit) until the final import is done.